tests/test_rabbitmq.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from testcontainers.rabbitmq import RabbitmqContainer
+import time
+import pika
+
+def test_declare_queue():
+    rabbitmq_container = RabbitmqContainer("rabbitmq:3-management")
+    with rabbitmq_container as rabbit:
+        time.sleep(30)
+        assert rabbit.declare_queue("prova")[1] == b'queue declared\n'
+
+
+def test_connection():
+    rabbitmq_container = RabbitmqContainer("rabbitmq:3-management")
+    with rabbitmq_container as rabbit:
+        time.sleep(30)
```

Comment:
In relation to #52, we probably want to follow the logs of the underlying rabbitmq container (e.g. using https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.logs) to wait until a startup message is emitted, e.g. I see `2020-01-10 17:30:54.380 [info] <0.8.0> Server startup complete; 0 plugins started.` in the logs of the container when it's ready.

Reply:
Thanks for the hint, I gonna try this way and add the other changes you suggested to the PR.